### ISSUE TITLE
DietPi-Software | Shairport-sync ARMv6 (Illegal instruction)

### ISSUE THREAD

**Issue by Fourdee** [MichaIng/DietPi#1548]
> https://twitter.com/WesBez/status/966469660597157888

**Comment by Fourdee** [comment]
32bit config example:
```
general =
{
  name = "%H";
  interpolation = "soxr";
};

metadata =
{
        enabled = "yes";
        include_cover_art = "no";
        pipe_name = "/tmp/shairport-sync-metadata";
        pipe_timeout = 5000;
        socket_address = "226.0.0.1";
        socket_port = 5555;
        socket_msglength = 65000;
};


alsa =
{
//  mixer_control_name = "PCM";
//  output_rate = 44100; // can be 44100, 88200, 176400 or 352800
  output_format = "S32"; // can be "U8", "S8", "S16", "S24", "S24_3LE", "S24_3BE" or "S32"
};
```

**Comment by Fourdee** [comment]
### I'll update all binaries to 3.1.7 (from 3.1.3), and, include x86_64.

#### Install/binary tests with update to 3.1.7 and recompiled:
```
root@DietPi:~# shairport-sync -v
alsa output device name is "default".
Version: "3.1.7-OpenSSL-Avahi-ALSA-soxr-metadata-sysconfdir:/usr/local/etc"
```

- 🈯️ ARMv6
- 🈯️ARMv7
- 🈯️ ARMv8
- 🈯️ x86_64

**Comment by Fourdee** [comment]
Resolved for v6.3.

### Solution for ARMv6 on v6.2 (copy and paste all):
```
wget http://dietpi.com/downloads/binaries/all/shairport-sync_3.1.7_armv6.7z -O package.7z
7z x -y package.7z -o/
rm package.7z
chmod +x /usr/local/bin/shairport-sync
dietpi-services restart
```
